core/src/main/java/com/hcl/ecomm/core/services/impl/CreateOrderServiceImpl.java

```diff
@@ -97,17 +97,16 @@ public JSONObject createOrderItem(JSONObject orderItem, String cartId, String cu
 				String output;
 
 				while ((output = br.readLine()) != null) {
-
 					order +=output;
 				}
 				JSONObject orderId = new JSONObject();
 
-				/*if(customerToken != null && !customerToken.isEmpty()) {*/
-				LOG.debug("Get Order Detail for order Id: ",order);
+				String order_Id=order.replaceAll("\\s", "").replaceAll("\"", "");
```

Comment:
@vishalgupta-hcl : why do we have 2 replace all methods here ?

String order_Id=order.replaceAll("\\s", "").replaceAll("\"", "");

Reply:
Hi @rishabh-behl , 
As you can see we are reading through each line and its appending every character like space as well as quote at line 100.
In order to replace all the space that got appended and all the quote from orderId we need to have 2 replace function.

Earlier: the order was _"23"
After:  we changed it to only 23(without space and quote)